features/install.feature

```diff
@@ -78,6 +78,29 @@ Feature: Installation
       And the file "config/initializers/localeapp.rb" should contain "config.api_key = ENV['LOCALEAPP_API_KEY']"
       And the exit status should be 0
 
+  Scenario: Running install with .env
```

Comment:
This scenario doesn't exist any more, instead there are scenarios `Saving api key in .env` and `Not saving api key in .env`; both of them now also "deal with arguments given to the `install` command".